### DISCUSSION TITLE
Help -  SMTP connect() failed.

### DISCUSSION THREAD

**Discussion by CarlosAlmeidaFF** [PHPMailer/PHPMailer#2557] [Q&A]
Hi There, 

I have this peace of code, pretty much standard. 

```php
<?php
// Import PHPMailer classes into the global namespace
// These must be at the top of your script, not inside a function
use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\SMTP;
use PHPMailer\PHPMailer\Exception;

// Load Composer's autoloader
require_once 'vendor/autoload.php';

    if($_SERVER["REQUEST_METHOD"] == "POST")
	{
       
		require_once("config.php"); 
        require_once("Code/functions.php");
		CheckSession();
        require_once("Code/ComposeReport.php");
	
		// Instantiation and passing `true` enables exceptions
		$mail = new PHPMailer(true);
		
		try{
			//Server settings
			$mail->SMTPDebug = 3;
			$mail->isSMTP();                                            
			$mail->Host       = 'my exchange server ip address';                   
			$mail->SMTPAuth   = true;                                   
			$mail->Username   = 'myuser@mydomain.com';                     
			$mail->Password   = mypassword';                               
			//$mail->SMTPSecure = 'tls';        
			$mail->Port       =  587;  
			
			//Recipients
			$mail->addAddress('destination@mydomain.com');            

			// Content
			$mail->isHTML(true);     
			// Set email format to HTML
			$mail->Subject = 'Here is the subject';
			$mail->Body    = 'This is the HTML message body <b>in bold!</b>';
			$mail->AltBody = 'This is the body in plain text for non-HTML mail clients';
			
			if ($mail->Send()) {
				$result = 1;
			} else {
				$result = "Error: " . $mail->ErrorInfo;
			}
		} catch (phpmailerException $e) {
			$result = $e->errorMessage();
		} catch (Exception $e) {
			$result = $e->getMessage();
		}
		
		$today = date("Y-m-d H:i:s"); 
		$myfile = fopen("output.txt", "a+") or die("Unable to open file!");
		
		$output = "[" . $today . "] " . $result ."\n";
		
		fwrite($myfile, $output);
		fclose($myfile);			
     }
?>
```

All I get is "SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting".
I've tried with a gmail account too and i get always the same error. :( 
I don't know more what to do to solve the issue.

I'm running Apache 2.4.50 on windows, php 5.6.9 and i installed the phpmailer manually and through Composer. 

Any thoughts?

Thank you in advance. 
Carlos

**Comment by Synchro** [comment]
First of all, you should not be using PHP 5.6. It's long out of support, and the version you're running has many known vulnerabilities.

Please post the SMTP transcript – with debug at level 3 you should at least see an "opening connection" line. If that's as far as it gets, then you need to read the troubleshooting guide as it's likely your hosting provider is blocking outbound SMTP.

**Reply by CarlosAlmeidaFF** [reply]
Hello @Synchro,

> Please post the SMTP transcript – with debug at level 3 you should at least see an "opening connection" line. If that's as far as it gets, then you need to read the troubleshooting guide as it's likely your hosting provider is blocking outbound SMTP.

Despite having changed the SMTPDebug value, the output error that is logged is always the same.

> First of all, you should not be using PHP 5.6.

It was working with a previous version of php, and phpmailer 5.5, i think.

**Comment by malvic-one** [comment]


**Reply by Synchro** [reply]
No, don't do that. There is no indication that certificate verification has anything to do with this problem, and even if it was it's the wrong approach as it's not solving the problem but hiding it and compromising your security.

**Comment by Nainavish15** [comment]
i'm also facing the same issue, while sending  mail through phpmailer.6.6.3 , what shoud i do, i tried many times

**Reply by Synchro** [reply]
[Read the docs](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#smtp-error-could-not-connect-to-smtp-host).

**Comment by Justice-Orogun** [comment]
I am having the same issue right now..... How can it be solved?

**Reply by Synchro** [reply]
[Read the docs](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#smtp-error-could-not-connect-to-smtp-host).
